iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -573,13 +573,13 @@ void LinalgBufferizePass::runOnFunction() {
     }
   });
   funcOp.walk([&](IREE::Flow::DispatchOutputStoreOp op) {
-    preProcessConvertInterfaceStoreTensorOp(b, op, bvm);
+    (void)preProcessConvertInterfaceStoreTensorOp(b, op, bvm);
```

Comment:
Hmmm we probably shouldn't be just casting these to void. There's a good reason this warns on unused results. Doesn't need to block the merge, since this is pre-existing, but probably should be fixed. @MaheshRavishankar 

Oh dear, we do this all over the place :-/

Reply:
yeah this is painful. THis needs to use Walk::Interrupt(). Inherited code. I'll see if I can fix it.

Reply:
Filed https://github.com/google/iree/issues/4759. This shouldn't block the merge